docs/gen-ai/llm-spans.md

```diff
@@ -0,0 +1,83 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: LLM Calls
+--->
+
+# Semantic Conventions for LLM requests
+
+**Status**: [Experimental][DocumentStatus]
+
+<!-- Re-generate TOC with `markdown-toc --no-first-h1 -i` -->
+
+<!-- toc -->
+
+- [Configuration](#configuration)
+- [LLM Request attributes](#llm-request-attributes)
+- [Events](#events)
+
+<!-- tocstop -->
+
+A request to an LLM is modeled as a span in a trace.
+
+The **span name** SHOULD be set to a low cardinality value representing the request made to an LLM.
+It MAY be a name of the API endpoint for the LLM being called.
+
+## Configuration
+
+Instrumentations for LLMs MUST offer the ability to turn off capture of prompts and completions. This is for three primary reasons:
```

Comment:
I think we should allow instrumentations that don't want to capture prompts and completions do their job.

```suggestion
Instrumentations MAY capture prompts and completions. Instrumentations that support it, MUST offer the ability to turn off capture of prompts and completions. This is for three primary reasons:
```

Reply:
Prefer using feature flags to turn this on and off, as with some LLMs/VectorDBs, different personas may have different requirements for this: Application Developer / Data Scientist may want to see prompt and response for debug purpose while end users sometime may want to hide those values for security reason, make sense? Thanks!

Reply:
> Prefer using feature flags to turn this on and off, as with some LLMs/VectorDBs, different personas may have different requirements for this: Application Developer / Data Scientist may want to see prompt and response for debug purpose while end users sometime may want to hide those values for security reason, make sense? Thanks!

The implementation will be up to the instrumentation library, but agree that this should be off by default and then turned on by some opt-in mechanism.